recipes/exchcxx/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "exchcxx" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/wavefunction91/ExchCXX/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 0ea38aab563ea5d11a03d80dc1bf909821091d903eb852c5cb350484753a847a
+
+build:
+  number: 0
+  skip: true  # [win]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x') }}
```

Comment:
Is there any docs on this upstream? Unfortunately now most projects do not care so much about ABI back-compatibility, if this project is an exception it is great, but it would be great if there is documentation upstream stating explicitly that they only break ABI on major version release.